### ISSUE TITLE
[Feature Request] Allow floating for new managed windows

### ISSUE THREAD

**Issue by diocletiann** [koekeishiya/yabai#2504]
I have custom placement logic for new windows, but new windows are immediately arranged with bsp and have to be moved, which causes a fair amount of noise and resizing. I tried making everything initially unmanaged but that creates with other issues. 

A setting to `float-new` would be perfect. As always, thanks for maintaining this amazing tool.

**Comment by Bellavene** [comment]
`yabai -m rule --add app=".*" manage=off`

**Comment by diocletiann** [comment]
> ```
> yabai -m rule --add app=".*" manage=off
> yabai -m rule --apply
> ```
> 
> or just use float layout if it suits your case.

I'm doing this now, but like I mentioned, it creates other issues. For example all managed windows become unmanaged when restarting the service.

**Comment by Bellavene** [comment]
You have not described correctly which config you have. So there is no way to understand your problem.

**Comment by 4ltayebb** [comment]
Maybe this is closer to what OP wants. My first thought is to add a signal for window creation to make the new window float and maybe make all other "older" floating windows managed. 

yabai -m signal --add event=<EVENT> action=<ACTION> [label=<LABEL>] [app[!]=<REGEX>] [title[!]=<REGEX>] [active=yes|no]

So you would have something like these two lines in your yabairc.
```shell
yabai -m rule --add label="manage_all" app=".*" manage=on
yabai -m signal --add event=window_created action='yabai -m rule --apply managed_all; yabai -m window $YABAI_WINDOW_ID --toggle float'
``` 
So this will have the effect of floating the new window only and making sure all other windows are managed by yabai. Of course you should exclude apps and windows you need to float all the time from this. 
Check relevant signal documentation [https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc#signal](here).

**Comment by diocletiann** [comment]
@4ltayebb I tried signals (and I use them to call into my tool that does the placement) but in this case with bsp on, the window gets placed (and others shuffle/resize) before the `--toggle float` takes effect.

**Comment by caferen** [comment]
It'd be great to somehow ensure a new window is never inserted into the tree to avoid the disruption. Ideally, I should still be able to apply other rules (e.g. grid).

**Comment by Bellavene** [comment]
I agree, that it would be awesome if he had an option to avoid the built-in bsp logic and could tell it where to place the window. I don't see a reason why it was not added at the beginning. Because there is anyway not more than 5 windows per screen opened usually. So why not script their placement order right in yabai at least if we have no other layout options?
